test/celeritas/optical/OpticalCollector.test.cc

```diff
@@ -336,12 +336,12 @@ TEST_F(LArSphereOffloadTest, TEST_IF_CELER_DEVICE(device_distributions))
 
     if (CELERITAS_REAL_TYPE == CELERITAS_REAL_TYPE_DOUBLE)
     {
-        EXPECT_EQ(3562272,
+        EXPECT_EQ(381086,
```

Comment:
Sorry, even though I thought about updating these I completely forgot by the time I merged 😞

Reply:
No worries, I forgot about the device tests too :)